src/devices/bus/heathzenith/h8/h8bus.h

```diff
@@ -0,0 +1,399 @@
+       *  -  Heath Company reserves the right to change these pin designations.
+
+
+    Signal         Pin       Direction        Description
+                             (wrt CPU board)
+    --------------------------------------------------------
+
+    A0-A15         30-45     Output           Address bus
+
+    D0-D7          10-17     Input/Output     Data bus
+
+    HOLD           27        Input            Hold - request CPU hold state
+
+    HLDA           25        Output           Hold Acknowledge - CPU has gone into hold status, and Address bus and
+                                              Data bus are in high impedance state
+
+    I/O R          26        Output           I/O Read
+
+    I/O W          21        Output           I/O Write
+
+    INT1-INT2      8-9       Input            Interrupt 1-2 (10, 20)
+
+    INT3-INT7      3-7       Input            Interrupt 3-7 (30, 40, 50, 60, 70)
+
+    M1             19        Output           Machine cycle 1 - CPU fetch cycle of the first byte of an instruction
+
+    MEMR           28        Output           Memory Read
+
+    MEMW           23        Output           Memory Write
+
+    ϕ2             22        Output           Phase 2 clock
+
+    RDYIN          20        Input            Ready Input - Asynchronous wait request to the clock generator, generates
+                                              the synchronous READY signal.
+
+    RESET          29        Input            Reset - holds CPU in reset, also resets the INTE and HLDA flip-flops
+
+    ROM DISABLE    46        Input            ROM Disable - Disable the on-board ROM
+
+    +18 V          47                         +18 V Unregulated
+    +8 V           48,49                       +8 V Unregulated
+    -18 V          2                          -18 V Unregulated
+    GND            0,1                        Ground
+    CPU GND        18,24                      CPU Ground
+
+
+***************************************************************************
+
+  P201 Cable
+
+  Between P1 and P2 was a cable labeled P201 in the schematics. This connected
+  the Front Panel(P1) card to the CPU board(P2), providing dedicated signal lines
+  between the two boards.
+
+    Signal         Pin       Direction        Description
+    --------------------------------------------------------
+
+    GND             1                         Ground
+    /INT2 (20)      2        P1 -> P2         Interrupt 20
+    /INTE           3        P2 -> P1         Interrupts enabled.
+    /INT1 (10)      4        P1 -> P2         Interrupt 10
+    /RESIN          5        P1 -> P2         Reset in, allowed keypad to initiate a reset of the computer
+
+***************************************************************************/
+
+#ifndef MAME_BUS_HEATHZENITH_H8_H8BUS_H
+#define MAME_BUS_HEATHZENITH_H8_H8BUS_H
+
+#pragma once
+#include "emu.h"
+#include <functional>
+#include <utility>
+#include <vector>
+#include <string.h>
```

Comment:
How many of these are actually used in this header?  It looks like `<utility>` and `<functional>` are used, but I can’t see anything from `<string.h>` at a glance.